tensorflow/lite/kernels/internal/conv_per_channel_quantized_test16x8.cc

```diff
@@ -0,0 +1,338 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Let's rename this file to conv_per_channel_quantized_16x8_test.cc

Reply:
thanks, renamed